src/panels/config/automation/action/ha-automation-action-row.ts

```diff
@@ -50,6 +52,8 @@ const OPTIONS = [
   "repeat",
   "choose",
   "device_id",
+  "stop",
+  "error",
```

Comment:
I would have expected these to be a single UI. "Error" is ambiguous and so I would name it something like "Stop with an error" or something. In that case, both are "Stop"

Reply:
Alright, should we adjust the YAML version too in that case?

Current:

```yaml
- stop: "Reason"
- error: "Reason"
```

Possible alternative:

```yaml
- stop:
    reason: "Reason"
    error: true
```

Reply:
Yeah let's do that.

Reply:
Alright, converting this one to draft to adjust the backend first

Reply:
Backend change: <https://github.com/home-assistant/core/pull/70109>

Reply:
Frontend adjust here: [4922c90](https://github.com/home-assistant/frontend/pull/12299/commits/4922c90e7a6686ea71bd3d35940a8a218248f4fc)